### PR TITLE
[skip ci] change external-network to public-network

### PR DESCRIPTION
**Pull request by caglar10ur**

### CODE REVIEW

File: tests/test-cases/Group12-VCH-BC/12-01-Delete.robot [hunk 1]
```diff
@@ -16,7 +16,7 @@ Install VIC 0.6.0 to Test Server
     Set Test Environment Variables
 
     Log To Console  \nInstalling VCH to test server...
-    ${output}=  Run  ./vic/vic-machine-linux create --debug 1 --name=${vch-name} --target=%{TEST_URL}%{TEST_DATACENTER} --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --appliance-iso=./vic/appliance.iso --bootstrap-iso=./vic/bootstrap.iso --password=%{TEST_PASSWORD} --bridge-network=%{BRIDGE_NETWORK} --external-network=%{PUBLIC_NETWORK} --compute-resource=%{TEST_RESOURCE} --timeout %{TEST_TIMEOUT}
+    ${output}=  Run  ./vic/vic-machine-linux create --debug 1 --name=${vch-name} --target=%{TEST_URL}%{TEST_DATACENTER} --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --appliance-iso=./vic/appliance.iso --bootstrap-iso=./vic/bootstrap.iso --password=%{TEST_PASSWORD} --bridge-network=%{BRIDGE_NETWORK} --public-network=%{PUBLIC_NETWORK} --compute-resource=%{TEST_RESOURCE} --timeout %{TEST_TIMEOUT}
```
Comment (hmahmood) [inline tests/test-cases/Group12-VCH-BC/12-01-Delete.robot]: This one does not need to change since it's running an older version of VIC.

**Comment by hmahmood** [comment]
lgtm except for that one change I pointed out.

File: tests/test-cases/Group12-VCH-BC/12-01-Delete.robot (hunk 1 above)
Comment (caglar10ur) [inline tests/test-cases/Group12-VCH-BC/12-01-Delete.robot]: ops, done

**Comment by mdubya66** [comment]
lgtm - don't forget to add the skip ci to the merge

**Pull request merged**